morphic.js

```diff
@@ -8047,22 +8131,45 @@ StringMorph.prototype.deleteSelection = function () {
 };
 
 StringMorph.prototype.selectAll = function () {
+    var cursor;
     if (this.isEditable) {
         this.startMark = 0;
+        cursor = this.root().cursor;
```

Comment:
You can never select more than one thing at the same time, as the cursor and selection both die whenever a `StringMorph` or a `TextMorph` loses its focus.

Additionally, placing the cursor at the end of the selection here makes the whole funny selection plus arrow key behavior much easier to deal with, as there's only one special case left to take care..